doc/source/serve/advanced-guides/multi-app-container.md

```diff
@@ -57,39 +59,41 @@ RUN sudo apt-get update && sudo apt-get install curl -y
 
 # Download the source code for the ResNet application into `resnet50_example.py`.
 RUN curl -O https://raw.githubusercontent.com/ray-project/ray/master/doc/source/serve/doc_code/resnet50_example.py
+
+ENV PYTHONPATH "${PYTHONPATH}:/home/ray"
```

Comment:
Could you add a comment explaining why this is necessary, for anyone who runs into this in the future?